scapy/contrib/rtcp.py

```diff
@@ -0,0 +1,110 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Pavel Oborin <oborin.p@gmail.com>
+# This program is published under a GPLv2 license
+
+# RFC 3550
+# scapy.contrib.description = Real-Time Transport Control Protocol
+# scapy.contrib.status = loads
+
+import struct
+
+from scapy.packet import Packet, bind_layers, Padding, Raw
+from scapy.fields import (
+    BitField, ByteEnumField, IntField, ConditionalField,
+    PacketListField, BitFieldLenField,
+    LongField, PacketField, ByteField,
+    X3BytesField, LenField, PacketLenField,
+    FieldListField, StrLenField, PadField,
+    FieldLenField
+)
+
+
+_rtcp_packet_types = {
```

Comment:
You could move this definition inside the RTCP Packet definition. This would be more clean

Reply:
TBH when they're private (`_`) it really doesn't matter.